providers/certification-client/units/client-cert-desktop-18-04.pxu

```diff
@@ -34,7 +34,6 @@ nested_part:
     misc-client-cert-manual
     mediacard-cert-manual
     memory-manual
-    mobilebroadband-cert-manual
```

Comment:
Since we should keep `wwan-manual`, you can replace this line with `wwan-manual`. This way, if manual WWAN tests are added in the future, they will be automatically part of the desktop test plans.